flax/nnx/summary.py

```diff
@@ -174,11 +178,27 @@ def _to_dummy_array(x):
     rows.append(call_info)
     out = method(obj, *args, **kwargs)
     call_info.outputs = jax.tree.map(_to_dummy_array, out)
+    if compute_flops:
+      call_info.flops = _get_flops(method, obj, *args, **kwargs)
+    if compute_vjp_flops:
+        call_info.vjp_flops = _get_vjp_flops(method, obj, *args, **kwargs)
     return out
 
   return method_wrapper
 
 
+def _get_flops(fn, *args, **kwargs):
+  e = nnx.jit(fn).lower(*args, **kwargs).compile()
+  cost = e.cost_analysis()
+  return 0 if cost is None or 'flops' not in cost else cost['flops']
+
+def _get_vjp_flops(fn, *args, **kwargs):
+  g = nnx.grad(fn)
```

Comment:
Please confirm how this is done in the linen implementation.

Reply:
Looks like linen supports arbitrary vjps, not just gradients. I'll make mine match.